darwin/DarwinMachine.c

```diff
@@ -75,6 +85,9 @@ void Machine_scan(Machine* super) {
    host->prev_load = host->curr_load;
    DarwinMachine_allocateCPULoadInfo(&host->curr_load);
    DarwinMachine_getVMStats(&host->vm_stats);
+#if defined(__arm64__)
+   DarwinMachine_getVMStats64(&host->vm_stats64);
```

Comment:
Any reason to duplicate the effort of retrieving both `vm_stats` structures?

Reply:
Just to prevent affecting existing functionality, but I checked the code again and it didn't affect anywhere else.
Same as above. I fix that.